documents/order.md

```diff
@@ -11,7 +11,7 @@ data := map[string]interface{}{
   "notes": map[string]interface{}{
       "key1": "value1",
       "key2": "value2",
-    } 
+    },
```

Comment:
Return NULL please check

Reply:
working from my side try this once 
`	data := map[string]interface{}{
		"amount": 100,
		"currency": "INR",
		"receipt": "@receipt 33",
		"partial_payment": false,
		"notes": map[string]interface{}{
			"key1": "value1",
			"key2": "value2",
		  },
	  }
	  body, err := client.Order.Create(data, nil)`